src/app/shared/header/case-header.component.ts

```diff
@@ -1,13 +1,32 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { CaseView } from '../../core/cases/case-view.model';
+import { CaseField } from '../domain/definition/case-field.model';
 
 @Component({
   selector: 'ccd-case-header',
-  templateUrl: './case-header.html'
+  templateUrl: './case-header.html',
+  styleUrls: ['./case-header.scss']
 })
-export class CaseHeaderComponent {
+export class CaseHeaderComponent implements OnInit {
 
   @Input()
   caseDetails: CaseView;
+  caseTitle: CaseField;
+  caseFields: CaseField[];
 
+  ngOnInit(): void {
+    this.caseTitle = new CaseField();
+    if (this.caseDetails.state.title_display) {
+      this.caseTitle.label = this.caseDetails.state.title_display;
+      this.caseFields = this.getCaseFields();
+    }
+  }
+
+  private getCaseFields(): CaseField[] {
```

Comment:
I think this method belongs to CaseView itself. Can we move it there?

Reply:
Tried moving but didn't work. 'Not a function' error. Not a straight forward one, so will leave it out.

Reply:
Will handle this as part of https://tools.hmcts.net/jira/browse/RDM-2681